android/manifest

```diff
@@ -15,4 +15,4 @@ name: ti.bluetooth
 moduleid: ti.bluetooth
 guid: 1d5352ed-3084-4f88-9f02-8e7be75f4ca9
 platform: android
-minsdk: 6.0.0.GA
+minsdk: 7.5.0
```

Comment:
make this 7.0.0? Shouldn't break anything and would make it more useful for people with existing apps.